lib/elixir/lib/kernel.ex

```diff
@@ -2805,9 +2805,10 @@ defmodule Kernel do
   ## Module names
 
   A module name can be any atom, but Elixir provides a special syntax which is
-  usually used for module names. What is called a module name is an upper case
-  character followed by any alphanumeric characters or `_` or `.`. This
-  identifier is equivilant to an atom prefixed by `Elixir.`. So in the
+  usually used for module names. What is called a module name is an _upper case
+  Latin letter_ followed by any number of _Latin letters_, _numbers_, _underscores_
```

Comment:
Well, i used "Latin" just to mean that, I know  what you mean.
in the [perl manual](http://perldoc.perl.org/perlrecharclass.html) they refers as "lowercase ASCII letter"